src/pages/resultsView/survival/SurvivalChart.tsx

```diff
@@ -192,11 +193,10 @@ export default class SurvivalChart extends React.Component<ISurvivalChartProps,
                             : {this.tooltipModel.datum.x.toFixed(2)} months {this.tooltipModel.datum.status ? "" :
                                 "(censored)"}
                         </div>
-
                     </Popover>
                 }
 
-                <table className="table table-striped" style={{marginTop:20}}>
+                <table className="table table-striped" style={{marginTop:20, width: "79%"}}>
```

Comment:
I would do with pixels because we may soon want to use a responsive layout in SPA